scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -449,6 +469,10 @@ Function PrepareTestFromArtifacts
 
                 (Get-Content $DeploymentWorkingFilePath).replace('<Analyzer.EventHubConnectionString>',$EventHubConnectionString) | Set-Content $DeploymentWorkingFilePath
                 (Get-Content $DeploymentWorkingFilePath).replace('<Analyzer.ConsumerGroupId>',$EventHubConsumerGroupId) | Set-Content $DeploymentWorkingFilePath
+                (Get-Content $DeploymentWorkingFilePath).replace('<Analyzer.LogAnalyticEnabled>',$AnalyzerLaEnabled) | Set-Content $DeploymentWorkingFilePath
+                (Get-Content $DeploymentWorkingFilePath).replace('<Analyzer.LogAnalyticWorkspaceId>',$AnalyzerLaWorkspaceId) | Set-Content $DeploymentWorkingFilePath
```

Comment:
any exception if $AnalyzerLaWorkspaceId is $null?

Reply:
Nope, I tested on powershell. It would simply delete the string.